From 1962edece401d76e56130e8e875ba7f7ac2335b3 Mon Sep 17 00:00:00 2001
From: Shunqian Zheng <zhengsq@rock-chips.com>
Date: Mon, 28 May 2018 08:53:31 +0800
Subject: [PATCH] ARM: dts: rk312x: add pcfg_output_high

This add a pcfg_output_high label so pinctl can set
a gpio output high by default in dts.

Change-Id: Ie4aa2dd59df3110b8e24d0a07cc9961ee086272a
Signed-off-by: Shunqian Zheng <zhengsq@rock-chips.com>
---
 arch/arm/boot/dts/rk312x.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/rk312x.dtsi b/arch/arm/boot/dts/rk312x.dtsi
index 1c5134d97d9e..faafbc0c6b44 100644
--- a/arch/arm/boot/dts/rk312x.dtsi
+++ b/arch/arm/boot/dts/rk312x.dtsi
@@ -1126,6 +1126,10 @@
 			bias-pull-pin-default;
 		};
 
+		pcfg_output_high: pcfg-output-high {
+			output-high;
+		};
+
 		pcfg_pull_none: pcfg-pull-none {
 			bias-disable;
 		};
-- 
2.35.3

